server/server.go

```diff
@@ -256,7 +258,7 @@ func (s *Server) Start() error {
 
 // initHTTP registers http prefixes.
 func (s *Server) initHTTP() {
-	s.mux.Handle(rpc.DefaultRPCPath, s.rpc)
+	s.rpc.SetDisabled(false)
 
 	s.mux.Handle("/", http.FileServer(
```

Comment:
By the same logic, if someone visits the admin UI while the node is bootstrapping, they would rather get a 503 instead of a 404 (or even better, a clear message of what's going on). Maybe should handle this at a higher level: can we add routes to `mux` that return 503s and later overwrite them with the real handlers, or would we need a layer of indirection to swap between the disabled and enabled handlers?